docs/resources/sso_default_roles.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "materialize_sso_default_roles Resource - terraform-provider-materialize"
+subcategory: ""
+description: |-
+  
+---
+
+# materialize_sso_default_roles (Resource)
+
+
+
+## Example Usage
+
+```terraform
+resource "materialize_sso_default_roles" "example_sso_default_roles" {
+  sso_config_id = materialize_sso_configuration.example_sso_config.id
```

Comment:
Can we be consistent about the naming of "config" vs "configuration"? I think either the resource should be called `sso_config` or this parameter should be called `sso_configuration_id`. Weak preference for the shorter name (`sso_config`) throughout.